assets/js/components/testimonial-list.js

```diff
@@ -0,0 +1,40 @@
+import BaseComponent from "./base-component.js";
+import State from '../state.js';
+import Tools from '../tools.js';
+
+class TestimonialList extends BaseComponent {
+  static rootSelector = ".testimonial-list";
+
+  constructor(root, options) {
+    super(root, options);
+    this.testimonialContainer = root.querySelector(".testimonial-list__contents");
+    this.testimonials = root.querySelectorAll(".testimonial-list__content");
+    this.start();
+  }
+
+  handleScrollEvent(){
+    this.testimonials.forEach((testimonial) => {
+      if (Tools.isInViewportPercent(testimonial, 30)) {
```

Comment:
Vielleicht kannst du die 30 noch in einer lokalen variable auslagern, damit wir keine magic numbers haben.